modules/apps/journal/journal-web/test/js/modals/ImportDataDefinitionModal.js

```diff
@@ -0,0 +1,212 @@
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {act, fireEvent, render, waitForElement} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import Modal from '../../../src/main/resources/META-INF/resources/js/modals/ImportDataDefinitionModal';
+
+const ImportDataDefinitionModal = () => <Modal portletNamespace="test" />;
+
+describe('Import Structure Modal', () => {
+	let container, file;
+
+	beforeAll(() => {
+		Liferay.component = jest.fn();
+		container = render(<ImportDataDefinitionModal />);
+		file = new File(['testing file upload'], 'Testing.lar', {
+			type: '.lar',
+		});
+	});
+
+	it('renders the title modal', async () => {
+		const {getByText} = container;
+
+		act(() => {
+			Liferay.component.mock.calls[0][1].open();
+		});
+
+		const title = await waitForElement(() => getByText('import-structure'));
+
+		expect(title).toBeInTheDocument(true);
+	});
+
+	it('renders the info alert', () => {
+		const {queryByText} = container;
+
+		const infoAlert = queryByText(
+			'the-import-process-will-run-in-the-background-and-may-take-a-few-minutes'
+		);
+
+		expect(infoAlert).toBeInTheDocument(true);
+	});
+
+	it('renders name input', () => {
+		const {queryByText} = container;
+
+		const inputName = queryByText('name');
+
+		expect(inputName).toBeInTheDocument(true);
+	});
+
+	it('input name has correct name prop', () => {
+		const {getByLabelText} = container;
+
+		const inputName = getByLabelText('name');
+
+		expect(inputName).toHaveAttribute('name', 'testname');
+	});
+
+	it('renders json input', () => {
+		const {queryByText} = container;
+
+		const jsonInput = queryByText('json-file');
+
+		expect(jsonInput).toBeInTheDocument(true);
+	});
+
+	it('input json file has correct name prop', () => {
+		const inputs = document.getElementsByName('testjsonFile');
+
+		expect(!!inputs).toBe(true);
+	});
+
+	it('renders cancel button in footer', () => {
+		const {queryByText} = container;
+
+		const buttonCancel = queryByText('cancel');
+
+		expect(buttonCancel).toBeInTheDocument(true);
+	});
+
+	it('the cancel button closes modal', () => {
+		const {queryByText} = container;
+
+		const buttonCancel = queryByText('cancel');
+
+		userEvent.click(buttonCancel);
+
+		const body = document.getElementsByTagName('body').item(0);
+
+		expect(body).toHaveAttribute('class', '');
+	});
+
+	it('renders import button in footer', () => {
+		const {queryByText} = container;
+
+		const buttonImport = queryByText('import');
+
+		expect(buttonImport).toBeInTheDocument(true);
```

Comment:
[toBeInTheDocument()](https://github.com/testing-library/jest-dom#tobeinthedocument) needs no parameters:
```suggestion
		expect(buttonImport).toBeInTheDocument();
```